openedx/core/djangoapps/credentials/migrations/0002_auto_20160325_0631.py

```diff
@@ -0,0 +1,18 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('credentials', '0001_initial'),
+    ]
+
+    operations = [
+        migrations.AlterModelOptions(
+            name='credentialsapiconfig',
+            options={},
```

Comment:
@ahsan-ul-haq This migration was missing for credentials, As you made most of the changes around this app, Does this addition makes sense to you?